docs/src/architecture/assets.md

```diff
@@ -0,0 +1,53 @@
+# Assets
+The Miden Rollup aims to provide a reliable protocol and software for users to exchange assets. In Miden, users can create and trade arbitrary fungible and non-fungible assets. 
```

Comment:
I would remove "and softare"

Reply:
Is already removed (after the rebase also here)